pkg/private/util.bzl

```diff
@@ -122,5 +122,7 @@ def get_repo_mapping_manifest(src):
     """
     files_to_run_provider = get_files_to_run_provider(src)
     if files_to_run_provider:
+        # repo_mapping_manifest may not exist in older Bazel versions (<7.0.0)
```

Comment:
Now "that" is a comment which makes it clear why you are doing something specal. Thanks.